src/main.controller.js

```diff
@@ -803,6 +819,71 @@ plugin.controller('wgnConfigCtrl', ['$scope', '$q', '$routeParams', 'znData', 'z
 			});
 		}
 
+		/**
+		 * This function will take a filter object and update all attribute and value properties
+		 * with the data from the config object.
+		 *
+		 * @param {Object} filter The filter to update
+		 * @param {Object} config The config to reference while updating
+		 *
+		 * @returns {Object} the updated (and now valid) filter
+		 */
+		function reconstructFilter (filter, config) {
+			let newFilter = angular.extend({}, filter);
```

Comment:
@tehpsalmist plz dont use `let` this code isnt being transpiled!

Reply:
Good catch! That solves my publishing problem, too!